Editor/Scripts/DCCIntegration/Launchers/BlenderLauncher.cs

```diff
@@ -121,6 +121,17 @@ public void OpenDCCTool(UnityEngine.Object asset)
             }
         }
 
+        public void CloseDCCTool() {
```

Comment:
Hm, hasn't this been merged in another PR ?

This is  also not related to this PR, but I'll approve this since it's just a few lines.